packages/core/src/middleware/koa-security-headers.ts

```diff
@@ -0,0 +1,49 @@
+import type { MiddlewareType } from 'koa';
+import compose from 'koa-compose';
+import helmet from 'koa-helmet';
+
+export default function koaSecurityHeaders<StateT, ContextT, ResponseBodyT>(): MiddlewareType<
+  StateT,
+  ContextT,
+  ResponseBodyT
+> {
+  return compose([
+    // Expect-ct: https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#expect-ct  not recommended
+    // helmet.expectCt(),
+
+    // X-Powered-By: https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#x-powered-by
+    helmet.hidePoweredBy(),
+
+    // Strict-Transport-Security: https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#strict-transport-security-hsts
+    helmet.hsts(),
+
+    // X-Download-Options
+    helmet.ieNoOpen(),
+
+    // X-Content-Type-Options: https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#x-content-type-options
+    helmet.noSniff(),
+
+    // X-Permitted-Cross-Domain-Policies
+    helmet.permittedCrossDomainPolicies(),
+
+    // Referrer-Policy https://cheatsheetseries.owasp.org/cheatsheets/HTTP_Headers_Cheat_Sheet.html#referrer-policy
+    helmet.referrerPolicy(),
```

Comment:
maybe `strict-origin-when-cross-origin` is good enough? i see by default it's `no-referrer` which is too strict.